app/src/main/java/nl/tudelft/b_b_w/view/PairActivity.java

```diff
@@ -0,0 +1,154 @@
+package nl.tudelft.b_b_w.view;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+
+import java.util.List;
+
+import nl.tudelft.b_b_w.R;
+import nl.tudelft.b_b_w.controller.BlockController;
+import nl.tudelft.b_b_w.model.Block;
+import nl.tudelft.b_b_w.model.BlockFactory;
+
+import static android.R.attr.id;
+
+/**
+ * This activity
+ */
+public class PairActivity extends Activity {
+    /**
+     * The block controller.
+     */
+    private BlockController blockController;
+
+    /**
+     * It's own block.
+     */
+    private Block block1;
+
+    /**
+     * a block.
+     */
+    private Block block2;
+
+    /**
+     * a block.
+     */
+    private Block block3;
+
+    /**
+     * a block.
+     */
+    private Block block4;
+
+
+    /**
+     * The name of the owner of each block in the chain.
+     */
+    private String ownerName;
+
+
+    /**
+     * The on create method sets up the activity.
+     * @param savedInstanceState brings the variables.
+     */
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_pair);
+    }
+
+
+    /**
+     * This method create the first test subject.We do this in order to simulate a transaction.
+     * @param view The view of the program.
+     */
+    public void onTestSubject1(View view) {
+        ownerName = "TestSubject1";
+        blockController = new BlockController(this);
+        block1 = BlockFactory.getBlock("BLOCK", ownerName, "HASH1","N/A","N/A","TestSubject_PUBKEY","IBANTestSubject1",0);
```

Comment:
To my understanding this is hardcoded explicitly for demo purposes to view how the pairing would work when paired with bluetooth.